grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -1013,28 +1015,27 @@ class OrderService {
 
     def getProductsInOrders(String[] terms, Location destination, Location vendor) {
         return OrderItem.createCriteria().list {
+            createAlias('product', 'prod', JoinType.LEFT_OUTER_JOIN)
+            createAlias('product.synonyms', 'syn', JoinType.LEFT_OUTER_JOIN)
+
             not {
                 'in'("orderItemStatusCode", OrderItemStatusCode.CANCELED)
             }
             order {
                 eq("destination", destination)
                 eq("origin", vendor)
             }
-            product {
-                if (terms) {
-                    terms.each { term ->
-                        term = term + "%"
-                        or {
-                            ilike("name", "%" + term)
-                            synonyms {
-                                and {
-                                    ilike("name", "%" + term)
-                                    eq("synonymTypeCode", SynonymTypeCode.DISPLAY_NAME)
-                                }
-                            }
-                            ilike("productCode", term)
-                            ilike("description", "%" + term)
+            if (terms) {
+                terms.each { term ->
+                    term = term + "%"
+                    or {
+                        ilike("prod.name", "%" + term)
+                        and {
+                            ilike("syn.name", "%" + term)
+                                eq("syn.synonymTypeCode", SynonymTypeCode.DISPLAY_NAME)
```

Comment:
could you fix the indent here, so it's not acting like it'd be nested inside `ilike`?